### PR TITLE
[Merged by Bors] - chore: flip and rename rank_eq_of_injective

### PR DESCRIPTION
**Pull request by kim-em**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/LinearAlgebra/Dimension.lean [hunk 1]
```diff
@@ -198,10 +198,10 @@ theorem LinearEquiv.rank_eq (f : M ≃ₗ[R] M₁) : Module.rank R M = Module.ra
   Cardinal.lift_inj.1 f.lift_rank_eq
 #align linear_equiv.rank_eq LinearEquiv.rank_eq
 
-theorem rank_eq_of_injective (f : M →ₗ[R] M₁) (h : Injective f) :
-    Module.rank R M = Module.rank R (LinearMap.range f) :=
-  (LinearEquiv.ofInjective f h).rank_eq
-#align rank_eq_of_injective rank_eq_of_injective
+theorem rank_range_of_injective (f : M →ₗ[R] M₁) (h : Injective f) :
```
Comment (eric-wieser) [inline Mathlib/LinearAlgebra/Dimension.lean:201]: Do we have the `finrank` version of this anywhere, and does it also need renaming?

**Review by eric-wieser — APPROVED** [review]
bors d+

Thanks

File: Mathlib/LinearAlgebra/Dimension.lean (hunk 1 above)
Comment (semorrison) [reply Mathlib/LinearAlgebra/Dimension.lean:201]: It exists as `finrank_range_of_inj`. I think the `_inj` vs `_injective` naming conundrum is out of scope for this PR. :-)

**Comment by semorrison** [comment]
bors merge

**Pull request closed without merging**